include/open62541/plugin/nodestore.h

```diff
@@ -411,6 +421,16 @@ struct UA_NodeHead {
 #endif
 };
 
+UA_LocalizedText UA_NodeStore_getDisplayNameForHead(const UA_NodeHead *head, UA_Server *server,
```

Comment:
This should be an internal API to reduce the API surface.
The user typically does not have access to any session ids.

In our method signature we typically go from "big context" to local context.
In that sense, use the server for the first argument, then the node-head, then the value.

Reply:
The user always gets the sessionId in callbacks (method, data source, etc...) so I thought it could be handy to have access to this